app/controllers/admin/authorised_email_domains_controller.rb

```diff
@@ -0,0 +1,25 @@
+class Admin::AuthorisedEmailDomainsController < AdminController
+  def index
+    @authorised_email_domains = AuthorisedEmailDomain.all
```

Comment:
Is this used to list all the allowed domains?

Reply:
Yup